apiserver/api_server.go

```diff
@@ -253,6 +254,39 @@ func saveBitriseYMLFromJSONHandler(w http.ResponseWriter, r *http.Request) {
 	respondWithJSON(w, 200, SimpleResponse{Message: "OK"})
 }
 
+func loadSpecHandler(w http.ResponseWriter, r *http.Request) {
+	// Load steplib spec
+	cmd := command.New("stepman", "collections", "--format", "json")
```

Comment:
maybe we should add a `stepman` command to `bitrise cli`, to expose it? so that `stepman collection ...` would be `bitrise stepman collection ...`?